documentation/ToolGroups/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: ToolGroups
+permalink: /tool_groups/
+nav_order: 370
+layout: page
+has_toc: false
+has_children: false
+---
+# ToolGroups
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+ToolGroups are used to group multiple Tools together.
+
+## ToolGroupSpecification Scheme
+
+| Property         | Default   | Required  | Description                                                                |
+|------------------|-----------|-----------|----------------------------------------------------------------------------|
+| Id               | -         | Yes       | Unique identifier                                                          |
+| GroupId          | -         | No        | Group unique identifier                                                    |
+| Type             | -         | Yes       | Defines what tool will be created                                          |
```

Comment:
Maybe atleast Type, Section and Layout would require info for what the possible valuer are, or is that documented somewhere already? I'd imagine it can be hard for new modders to know what the possibilities are, when they can only see 1 example value

Reply:
Added a link to the folder with all the tool/groups for always up to date information. Or that they need to go to the mod creator.